scripts/static_text/meettheteam/content_meettheteam_table0_row0col0.js

```diff
@@ -1 +1 @@
-$("#id_meettheteam_table0_row0col0").append("<img id='id_meettheteam_table0_row0col0_img0' src='../Neurobi.github.io/assets/images/MRExpertThumbnail3-150x150.png' alt='VR Expert'>").append("<br><br>Meet our expert of all things virtual, mixed and augmented reality. Steve has a passion for visual interaction, with a decade of graphics & 3D modeling experience for serious games design brought to life using solid coding credentials and an eye for detail. Having spent the first half of his career in Computer Engineering, his backend dev & hardware insights span the breadth of any project. Some of his current works are available for purchase in the <a style='color:#00ccff' href='https://www.assetstore.unity3d.com/#!/content/142648'><strong>Unity Asset Store</strong></a>.<br><br><strong>Tags:</strong><br>Graphic Design, Data Visualization, Unity2D/3D, C#, JS<br><br><strong>Steve Biggs</strong>")
\ No newline at end of file
+$("#id_meettheteam_table0_row0col0").append("<img id='id_meettheteam_table0_row0col0_img0' src='../Neurobi.github.io/assets/images/MRExpertThumbnail3-150x150.png' alt='VR Expert'>").append("<br><br>Meet our expert of all things virtual, mixed and augmented reality. Steve has a passion for visual interaction, with a decade of serious games design experience brought to life using solid coding credentials and an eye for detail. Having started his career in Computer Engineering, his backend dev & hardware insights span the breadth of any project. <a style='color:#00ccff' href='https://upwardskystudios.com/portfolio.html'><strong>UpwardSkyStudios</strong></a>.<br><br><strong>Tags:</strong><br>Data Visualization, Graphics, Coding, 3D Modeling, Unity3D<br><br><strong>Steve Biggs</strong>")
```

Comment:
Looks good!

Reply:
(Review comment 4) I saw the open issue with inserting HTML via .txt files. Even though the content is mostly raw text, it's still technically HTML and I think for readability it would be easier for us to be able to split it up into multiple lines. This file is representative of how we handle all of our content currently, so this change would propagate to potentially many files